driver/tke_driver.go

```diff
@@ -296,36 +297,68 @@ func (d *Driver) GetDriverUpdateOptions(ctx context.Context) (*types.DriverFlags
 	driverFlag := types.DriverFlags{
 		Options: make(map[string]*types.Flag),
 	}
+	driverFlag.Options["secret-id"] = &types.Flag{
+		Type:  types.StringType,
+		Usage: "The secretID of the cluster",
+	}
+	driverFlag.Options["secret-key"] = &types.Flag{
+		Type:     types.StringType,
+		Password: true,
+		Usage:    "The secretKey of the cluster",
+	}
 	driverFlag.Options["cluster-id"] = &types.Flag{
 		Type:  types.StringType,
 		Usage: "The ID of the existing cluster",
 	}
-	driverFlag.Options["cluster-name"] = &types.Flag{
+	driverFlag.Options["node-count"] = &types.Flag{
+		Type:  types.IntType,
+		Usage: "The number of nodes purchased, up to 100",
+	}
+	driverFlag.Options["instance-type"] = &types.Flag{
 		Type:  types.StringType,
-		Usage: "The name of the cluster that should be displayed to the user",
+		Usage: "See CVM Instance Configuration for details . Default: S2.MEDIUM4",
 	}
-	driverFlag.Options["vpc-id"] = &types.Flag{
+	driverFlag.Options["storage-size"] = &types.Flag{
+		Type:  types.IntType,
+		Usage: "Data disk size (GB), the step size is 10",
+	}
+	driverFlag.Options["root-size"] = &types.Flag{
+		Type:  types.IntType,
+		Usage: "System disk size. Linux system adjustment range is 20 - 50G, step size is 1",
+	}
+	driverFlag.Options["cluster-name"] = &types.Flag{
 		Type:  types.StringType,
-		Usage: "The private vpc id the cluster",
+		Usage: "The name of the cluster that should be displayed to the user",
 	}
 	driverFlag.Options["cluster-desc"] = &types.Flag{
 		Type:  types.StringType,
 		Usage: "The description of the cluster",
 	}
-	driverFlag.Options["secret-id"] = &types.Flag{
+	driverFlag.Options["project-id"] = &types.Flag{
+		Type:  types.IntType,
+		Usage: "The ID of your project to use when creating a cluster",
+	}
+	driverFlag.Options["bandwidth-type"] = &types.Flag{
 		Type:  types.StringType,
-		Usage: "The secretID of the cluster",
+		Usage: "Type of bandwidth, PayByTraffic or PayByHour",
```

Comment:
Please add default value message to `Usage` in order to tell users which one is using if blank. Or you can just add default value for this flag.